README.md

```diff
@@ -43,4 +43,87 @@ If you'd like to try out the plugin pre-release, you can build it from source an
 
 ## Getting started
 
+The plugin can work in any of the  [JetBrains family of IDEs](https://www.jetbrains.com/products.html), let’s try the first example with Kubernetes and Java in IntelliJ IDE (the plugin supports both Community and Ultimate editions). Follow the installation steps above to install the plugin. Restart your IDE if prompted to activate the plugin.
```

Comment:
I wonder, if instead of having the full getting started here on the main page, we migrate them to the actual sample projects.

And instead, here we just link to each guide (with this one being the first); something along the lines of:

```
To get started using the plugin and developing on Kubernetes, see our example projects (link to the example root):
-  `hello-spring-boot` (link to the hello-spring-boot example) - a simple getting started project for Java and Spring boot
- `[future-sample-here]`
```

Then just move the rest of the guide to the `hello-spring-boot` page.

Reply:
I skimmed though a few Google Container tools repositories including Skaffold and kaniko, and some other popular repositories and in general it seems most of them put getting started first steps instructions on the main repo page, even if they are quite long. Some of the more popular repositories link to their documentation, but this is mostly for complex products with their separate own doc sites.

I feel it would be slightly more visible to keep getting started on this page, and move it if we have a lot more general information later - for now we don't have anything else. Otherwise visitors might just not visit this page at all.

Reply:
sounds good. sorry for the delay - picking up the review now

Reply:
some nits:

"The plugin can work" -> "The plugin works"
", let's try" -> ". The following shows an example using Kubernetes with Java and Spring Boot in IntelliJ IDEA (Community or Ultimate editions)."